spec/unit/spotlight/dor/indexer_spec.rb

```diff
@@ -25,27 +28,211 @@
       'SC0340, Accession 2005-101',
       'SC0340, Accession 2005-101, Box 18'
     ]
-    parsable_exemplars_based_on_actual_data.each do |example|
-      it "parses series number from '#{example}'" do
-        ng_mods = Nokogiri::XML("#{mods_start}#{example}#{mods_end}")
+    exemplars_based_on_feigenbaum.each do |example|
```

Comment:
> I wonder if we want to make a separate PR for the refactoring so this can get "in" as a working thing first, and refactored after, as an additional task.

We can talk about this after stand-up. Right now, I'd prefer to get it cleaned up now (past experience has shown we're not /that/ good at revisiting technical debt). Because of your work, we have several examples of what we're trying to do, and that's usually been a good time to extract common patterns. Because of the complexity of the methods we're testing, having a succinct test suite seems especially important if we need to revisit the code (and, given the various permutations you're accounting for here, I'm sure we'll need to tweak this in the future).